### PR TITLE
Added a haiku " Song of Things"

### PR DESCRIPTION
**Pull request by Mugdha-Hazra**
Hello everyone,
I'm adding a haiku as an open-source contribution please review it and do the needful.

### CODE REVIEW

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this haiku! There are some changes that need to be made to the syllable count in order to have a valid haiku. The test suite on our haiku will let you know which lines are failing, please check the **details** under "All checks have failed" of this pull request to figure out which lines need to be modified.

A haiku is composed of 3 lines. The first line is 5 syllables long, the second line is 7 syllables long, and the third line is 5 syllables long. Please feel free to use any human language.

For your haiku, the third line, `its mighty Mississippi` could be the second line instead as that is the correct number of syllables.

Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) if you need extra guidance. Once you amend this pull request, we can merge it in.

*Please note: after inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by Mugdha-Hazra** [comment]
Thank you soo much @ltagliaferri sir/madam for your valuable advice. I have made the necessary changes can you please review it again.

**Comment by ltagliaferri** [comment]
Awesome, this is great, thank you for revising and following through on this. So glad to have you as part of open source 🎉

**Review by ltagliaferri — APPROVED** [review]

**Pull request merged**
